### PR TITLE
Env

### PR DESCRIPTION
**Pull request by ahmad3213**
Combine new version

### CODE REVIEW

File: config/Run3_2022/samples.yaml [hunk 1]
```diff
@@ -12,6 +12,5 @@ DYto2L_M-50_madgraphMLM:
   crossSection: DY_NNLO
   crossSectionStitch: DY_NLO
   isReference: true
-
 SingleMuon_Run2022C:
-  sampleType: data
```
Comment (kandrosov) [inline config/Run3_2022/samples.yaml:17]: undo modifications to this file

File: env.sh [hunk 2]
```diff
@@ -99,7 +99,7 @@ EOF
   run_cmd cd "$cmssw_base/src"
   run_cmd eval `scramv1 runtime -sh`
   if ! [ -d "CombineHarvester" ]; then
-    run_cmd bash <(curl -s https://raw.githubusercontent.com/cms-analysis/CombineHarvester/main/CombineTools/scripts/sparse-checkout-ssh.sh)
+    git clone https://github.com/cms-analysis/CombineHarvester.git CombineHarvester
```
Comment (kandrosov) [inline env.sh:102]:
```suggestion
    run_cmd git clone https://github.com/cms-analysis/CombineHarvester.git CombineHarvester
```

File: env.sh [hunk 3]
```diff
@@ -180,26 +180,23 @@ action() {
...
   fi
-
   source /afs/cern.ch/work/k/kandroso/public/flaf_env/bin/activate
   source /cvmfs/sft.cern.ch/lcg/app/releases/ROOT/6.30.06/x86_64-almalinux9.3-gcc114-opt/bin/thisroot.sh
   run_cmd source /afs/cern.ch/user/m/mrieger/public/law_sw/setup.sh
   source "$( law completion )"
   source /cvmfs/cms.cern.ch/rucio/setup-py3.sh &> /dev/null
-
   alias cmsEnv="env -i HOME=$HOME ANALYSIS_PATH=$ANALYSIS_PATH ANALYSIS_DATA_PATH=$ANALYSIS_DATA_PATH X509_USER_PROXY=$X509_USER_PROXY CENTRAL_STORAGE=$CENTRAL_STORAGE ANALYSIS_BIG_DATA_PATH=$ANALYSIS_BIG_DATA_PATH DEFAULT_CMSSW_BASE=$DEFAULT_CMSSW_BASE DEFAULT_CMSSW_ARCH=$DEFAULT_CMSSW_ARCH $ANALYSIS_PATH/RunKit/cmsEnv.sh"
   alias cmbEnv="env -i HOME=$HOME ANALYSIS_PATH=$ANALYSIS_PATH ANALYSIS_DATA_PATH=$ANALYSIS_DATA_PATH X509_USER_PROXY=$X509_USER_PROXY CENTRAL_STORAGE=$CENTRAL_STORAGE ANALYSIS_BIG_DATA_PATH=$ANALYSIS_BIG_DATA_PATH /cvmfs/cms.cern.ch/common/cmssw-cc7 -- $ANALYSIS_PATH/cmb_env.sh"
```
Comment (kandrosov) [inline env.sh:201]:
```suggestion
  alias cmbEnv="env -i HOME=$HOME ANALYSIS_PATH=$ANALYSIS_PATH ANALYSIS_DATA_PATH=$ANALYSIS_DATA_PATH X509_USER_PROXY=$X509_USER_PROXY CENTRAL_STORAGE=$CENTRAL_STORAGE ANALYSIS_BIG_DATA_PATH=$ANALYSIS_BIG_DATA_PATH $ANALYSIS_PATH/cmb_env.sh"
```

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Comment by kandrosov** [comment]
@ahmad3213, please resolve conflicts and undo changes to config/Run3_2022/samples.yaml. Once done, the PR is ready to be merged. Thank you!

**Review by aebid — APPROVED** [review]
After cloning, adding this remote, getting this branch, and running source env.sh, I was able to generate the limit plot using previous binning.

**Comment by ahmad3213** [comment]
i can open a new PR with exact changes

**Pull request closed without merging**